### ISSUE TITLE
_sp JAR import was broken

### ISSUE THREAD

**Issue by bryaakov** [thomsonreuters/CM-Well#408]
for example: when importing `fuzzy-match.jar` which located in `/meta/lib`, the response is "Could not process request", and this shows in logfiles:

```
2018-01-21 12:40:15,028 lication-akka.actor.default-dispatcher-3644] ERROR controllers.SpHandlerController      - _sp failure
java.util.concurrent.ExecutionException: Boxed Error
        at scala.concurrent.impl.Promise$.resolver(Promise.scala:83)
        at scala.concurrent.impl.Promise$.scala$concurrent$impl$Promise$$resolveTry(Promise.scala:75)
        at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:280)
        at scala.concurrent.Promise.complete(Promise.scala:49)
        at scala.concurrent.Promise.complete$(Promise.scala:48)
        at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:183)
        at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
        at scala.concurrent.impl.ExecutionContextImpl$AdaptedForkJoinTask.exec(ExecutionContextImpl.scala:140)
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
        at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: com.google.common.util.concurrent.ExecutionError: java.lang.NoClassDefFoundError: com/hp/hpl/jena/sparql/function/FunctionBase2
        at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2216)
        at com.google.common.cache.LocalCache.get(LocalCache.java:4147)
        at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:4151)
        at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:5140)
        at cmwell.util.collections.package$LoadingCacheExtensions.$anonfun$getAsync$3(package.scala:462)
        at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
        at scala.concurrent.impl.ExecutionContextImpl$DefaultThreadFactory$$anon$2$$anon$5.block(ExecutionContextImpl.scala:76)
        at java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3313)
        at scala.concurrent.impl.ExecutionContextImpl$DefaultThreadFactory$$anon$2.blockOn(ExecutionContextImpl.scala:71)
        at scala.concurrent.package$.blocking(package.scala:142)
        at cmwell.util.collections.package$LoadingCacheExtensions.$anonfun$getAsync$2(package.scala:462)
        at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:655)
        at scala.util.Success.$anonfun$map$1(Try.scala:251)
        at scala.util.Success.map(Try.scala:209)
        at scala.concurrent.Future.$anonfun$map$1(Future.scala:289)
        at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
        ... 7 common frames omitted
```

**Comment by bryaakov** [comment]
Perhaps `fuzzy-match.jar` is compiled with an out-of-date Jena version?

**Comment by hochgi** [comment]
Will be resolved as part of #425 
A Reminder:
initialization process of crashable worker included an eager instantiation of a singleton class that should load and cache jars/sources/etc'...
the data needed to be loaded from either new or old path.
We picked old, knowing that for a very short period of time during migration, this functionality won't be available.

**Comment by GuyGuterman** [comment]
Tested:

Action | Expected | Actual
-- | -- | --
localhost | uploaded jar then ran query | as expected
Environment | uploaded jar then ran query | as expected

Moving to Done.
